test/Microsoft.IdentityModel.Tokens.Saml.Tests/SamlSecurityTokenHandlerTests.cs

```diff
@@ -185,6 +186,8 @@ public void ReadToken(SamlTheoryData theoryData)
             {
                 theoryData.ExpectedException.ProcessException(ex);
```

Comment:
You need to pass context to: ProcessNoException AND ProcessException.
ProcessNoException(context), ProcessException(ex, context)